### PR TITLE
Perf: add launch events

### PR DESCRIPTION
**Pull request by EdmondChuiHW**
# Summary

TLDR: two basic events for launch time

Open source: no-op as perf metrics is only applicable for internal builds. See https://github.com/facebookexperimental/rn-chrome-devtools-frontend/pull/16

* `Entrypoint.LoadingStarted`: fired when entry point begins loading
* `Debugger.IsReadyToPause`: fired when SDK is ready to pause

[Corresponding internal diff](https://www.internalfb.com/diff/D55164679)

# Stack

Stacked on #26 and #27

Preview this PR in isolation at https://github.com/EdmondChuiHW/rn-chrome-devtools-frontend2/pull/2/files

# Upstreaming plan

- [ ] This commit should be sent as a patch to the upstream `devtools-frontend` repo. I've reviewed the [contribution guide](https://docs.google.com/document/d/1WNF-KqRSzPLUUfZqQG5AFeU_Ll8TfWYcJasa_XGf7ro/edit#heading=h.9kj7femz1xg5).
- [x] This commit is React Native-specific and cannot be upstreamed.

### CODE REVIEW

**Review by huntie — APPROVED** [review]
LGTM :) (with blocking feedback in prior PRs)
